app/frontend/entrypoints/application.js

```diff
@@ -12,3 +13,16 @@ document
   })
 
 initAll()
+
+// We set rawAttribute to true to pass the value of the text input to the
+// controller when the autocomplete is being used, so we can detect when the
+// user has cleared the field. We set source to false to overide the DfE
+// component's handling of sort order and synonyms, as described in the code
+// found in
+// https://github.com/DFE-Digital/dfe-autocomplete/blob/36d80e6b5bba67c92cd9ec6982a4e536d1889aed/src/dfe-autocomplete.js#L54C1-L54C1
+// this means the autocomplete will get options straight from the select
+// element, removing the null option, which isn't needed for autocomplete.
+dfeAutocomplete({
+  rawAttribute: true,
+  source: false
```

Comment:
This is needed to stop the autocomplete including "Select an organisation" from the list of options. To set no organisation, the user needs to leave the autocomplete blank.

Reply:
Awesome, thanks for figuring this out 🙌

Reply:
I wonder if we should raise an issue upstream suggesting that the default `source` for dfe-autocomplete filters our null valued options?

Reply:
yeah - I don't think it would take away anything from their current behaviour. It might just be that they never need empty ones and so it hasn't bothered them. I don't think the code is used that widely.